mina-payout-reports/templates/service.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ include "mina-payout-reports.fullname" . }}
+  labels:
+    {{- include "mina-payout-reports.labels" . | nindent 4 }}
+spec:
+  type: {{ .Values.service.type }}
```

Comment:
**suggestion:** This makes the chart unable to render given that `service.type` value is not declared.

Reply:
in values they are already split per deployment. This is a mistake that still needed to be spotted :pray: